website/themes/docdock/exampleSite/content/content-organisation/_index.md

```diff
@@ -46,7 +46,7 @@ in the page frontmatter, add a `head` param to insert any HTML code before the m
 example to display a "Hello"
 
 	+++
-	title = "Github repo"
```

Comment:
This file and the rest of the files under this path belong to a sample website which comes with the Hugo template we're using for MetalLB. I wouldn't touch them because they come from [upstream](https://github.com/vjeantet/hugo-theme-docdock/blob/master/exampleSite/content/content-organisation/_index.md) and in any case aren't used.

Reply:
Thankyou for review, i fixed it.